src/ddmd/func.d

```diff
@@ -3189,6 +3182,28 @@ extern (C++) final class UnitTestDeclaration : FuncDeclaration
         return FuncDeclaration.syntaxCopy(utd);
     }
 
+    void setIdentifier() {
+        ident = createIdentifier(loc, _scope);
+    }
+
+    /***********************************************************
+     * Generate unique unittest function Id so we can have multiple
+     * instances per module.
+     */
+    private Identifier createIdentifier(Loc loc, Scope* sc) {
```

Comment:
Also the coding convention should be adhered to - curly braces on next line.

Reply:
Oops, fixing it.